### DISCUSSION TITLE
Repository template for model publishing

### DISCUSSION THREAD

**Discussion by daniel-dona** [k2-fsa/icefall#1608] [Q&A] [answered]
Is there any template/reference for publishing icefall trained models in places like Hugging Face?

Specially for folders and files, and what exports to include (onnx, ncnn?). But also for model cards, eval results, etc...

**Comment by csukuangfj** [comment, marked as answer]
Please refer to
https://github.com/k2-fsa/icefall/blob/master/egs/librispeech/ASR/RESULTS.md

You can find many models in the above RESULTS.md

You can select one and check what files need to be uploaded.
